packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -396,6 +402,15 @@ const ProductList = (props) => {
         }
     }, [productSearchResult])
 
+    /**************** Page Designer ****************/
+    const PAGEDESIGNER_TO_COMPONENT = {
+        'commerce_assets.productListTile': ImageWithText,
```

Comment:
for now i used the existing `ImageWithText` component but if we want to customize the banner I can create a separate component for it!